config/global.php

```diff
@@ -166,8 +166,15 @@
                 $ipsResolved[] = $ip;
             } else {
                 $ipFromHost = @gethostbyname($ip);
-                if (!empty($ipFromHost)) {
+                if (!empty($ipFromHost)
+                    && filter_var($ipFromHost, FILTER_VALIDATE_IP)) {
                     $ipsResolved[] = $ipFromHost;
+                } elseif (function_exists('dns_get_record')) {
```

Comment:
wouldn't it generally make sense to do always both. A lookup for IPv4 and IPv6. I guess a hostname could actually resolve both, right?